docs/components/data_tables.md

```diff
@@ -0,0 +1,407 @@
+<!--docs:
+title: "Data tables"
+layout: detail
+section: components
+excerpt: "Data tables display information in a grid-like format of rows and columns."
+iconId: 
+path: /catalog/data-tables/
+api_doc_root: true
+-->
+
+# Data tables
+
+[Data tables](https://material.io/components/data-tables/#) display sets of data across rows and columns.
+
+![Data table with 5 rows: 1 header row, 4 rows, one column of checkboxes](assets/data_tables/data-table-hero.png)
+
+## Using data tables
+
+Data tables display information in a grid-like format of rows and columns. They organize information in a way that’s easy to scan, so that users can look for patterns and insights.
+
+Data tables can contain:
+
+* Interactive components (such as chips, buttons, or menus)
+* Non-interactive elements
+* Tools to query and manipulate data
+
+### Making data tables accessible
+
+Flutter's text field component APIs supports both label text and helper text for accessibility.
```

Comment:
This message should be updated for data tables, looks like its for text field

Reply:
Done